test/apicoverage-webhook/apicoverage-webhook.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 The Knative Authors
```

Comment:
We would be installing this webhook only for a specific prow job. So it makes more sense to have everything confined to this folder and not install it when `test/config` is applied.  With regards to namespace, I don't know how it plays with validation webhook residing in different namespace as to one where objects are created. I will work on this as an improvement rather than changing it now if it is not too important.